IdentityCore/src/requests/MSIDSilentTokenRequest.m

```diff
@@ -166,7 +166,7 @@ - (void)executeRequestImpl:(MSIDRequestCompletionBlock)completionBlock
             MSID_LOG_WITH_CTX(MSIDLogLevelInfo, self.requestParameters, @"Found valid access token.");
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->